jerry-core/ecma/operations/ecma-objects-general.c

```diff
@@ -760,7 +780,7 @@ ecma_op_general_object_define_own_property (ecma_object_t *obj_p, /**< the objec
                                                        NULL,
                                                        NULL,
                                                        was_enumerable,
-                                                       was_configurable);
+                                                       is_current_configurable);
```

Comment:
Because of the early return condition above (at step 9), don't we know that is_current_configurable is true for sure?